File: C/PRAK205-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>
#include <math.h>

int main(){
    int a, b, c, luas, keliling;

    printf("Masukan nilai A =");
    scanf("%d", &a);
    printf("Masukan nilai B =");
    scanf("%d", &b);
    c = sqrt(b * b - a * a);
    luas = 0.5 * c * a;
    keliling = a + b + c;
    printf("\nOUTPUT\n");
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    printf("keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2", luas);

    return 0;
}

File: Python/PRAK205-2210817310009-Mario Franca Wijaya.py
import math
print("INPUT")
a = int(input("Masukan Nilai a\t:"))
b = int(input("Masukan Nilai b\t:"))
c = math.sqrt(b * b - a * a)
luas = 0.5 * c * a
keliling = a + b + c
print("OUTPUT")
print("Alas = {:.0f} cm".format(c))
print("Tinggi = {:.0f} cm".format(a))
print("Keliling = {:.0f} cm".format(keliling))
print("Luas = {:.0f} cm^2".format(luas))